data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/dlq/FailedDlqData.java

```diff
@@ -0,0 +1,120 @@
+package org.opensearch.dataprepper.plugins.sink.opensearch.dlq;
+
+import java.util.Objects;
+
+public class FailedDlqData {
```

Comment:
Do we not need the OpenSearch domain name ("hosts" config) here? For customer looking at this file, they would want to know which OS cluster is causing the problem?

Reply:
The plugin ID should help identify the host in the pipeline configuration. We have concerns about injecting the actual URI into the object as this could introduce a security concern during replay.